vnext/ReactUWP/Views/Image/ReactImage.cpp

```diff
@@ -52,6 +53,48 @@ void ReactImage::OnLoadEnd(winrt::event_token const &token) noexcept {
   m_onLoadEndEvent.remove(token);
 }
 
+void ReactImage::ResizeMode(react::uwp::ResizeMode value) {
+  if (m_resizeMode != value) {
+    m_resizeMode = value;
+
+    bool switchBrushes{false};
+
+    if (m_useCompositionBrush != ShouldUseCompositionBrush()) {
+      switchBrushes = true;
+      m_useCompositionBrush = ShouldUseCompositionBrush();
+    }
+
+    if (switchBrushes) {
+      SetBackground(m_memoryStream != nullptr, false);
+    } else {
+      const auto bitmapBrush{Background().as<winrt::ImageBrush>()};
+      bitmapBrush.Stretch(ResizeModeToStretch(m_resizeMode));
+    }
+  }
+}
+
+bool ReactImage::ShouldUseCompositionBrush() {
```

Comment:
>ShouldUseCompositionBrush [](start = 17, length = 25)

Looks overkill to define a function for a simple comparison, you can just do " bool shouldUseCompositionBrush = (m_resizeMode == ResizeMode::Repeat);" in ResizeMode, and you can also avoid calling ShouldUseCompositionBrush() twice too.